tip-wlan/charts/wlan-ssc-service/values.yaml

```diff
@@ -107,8 +107,8 @@ affinity: {}
 creds:
   enabled: true
   sslKeyPassword: DUMMY_PASSWORD
-  sslKeystorePassword: DUMMY_PASSWORD
-  sslTruststorePassword: DUMMY_PASSWORD
+  sslKeystorePassword: mypassword
+  sslTruststorePassword: mypassword
```

Comment:
Please revert this to DUMMY_PASSWORD, (until we change this file into a ConfigMap/Secret and read it from override file)